contracts/SaveDAI.sol

```diff
@@ -4,22 +4,25 @@ pragma solidity ^0.5.0;
 import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
 import "@openzeppelin/contracts/token/ERC20/ERC20Detailed.sol";
 import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
-import "@openzeppelin/contracts/token/ERC20/ERC20Pausable.sol";
+import "@openzeppelin/contracts/lifecycle/Pausable.sol";
 import "@openzeppelin/contracts/math/SafeMath.sol";
 import "@openzeppelin/contracts/ownership/Ownable.sol";
 import "./lib/UniswapExchangeInterface.sol";
 import "./lib/UniswapFactoryInterface.sol";
 import "./lib/CTokenInterface.sol";
 import "./lib/OTokenInterface.sol";
 
-contract SaveDAI is ERC20, ERC20Detailed, ERC20Pausable, Ownable {
+contract SaveDAI is ERC20, ERC20Detailed, Pausable, Ownable {
     using SafeMath for uint256;
 
     /***************
     GLOBAL CONSTANTS
     ***************/
     // Variable to set distant deadline for Uniswap tokenToTokenSwap transactions
-    uint256 constant LARGE_BLOCK_SIZE = 1651753129000;
+    uint256 constant LARGE_BLOCK_SIZE = 1099511627776;
```

Comment:
@dougiebuckets I thought he meant something like: `1651753129000**2`

Reply:
@kseniya292 I thought he was referring to the value. I.e., the value must be a power of two (i.e., 2^n) in order to optimize the 256-bit storage. If we use the `**` notation won't that require additional computation and thus cost more gas?

Reply:
I'm fairly certain he meant 2^n so that the value in binary is `1` followed by a bunch of zeroes, the latter of which [cost less gas to store](https://github.com/wolflo/evm-opcodes#a6-sstore)

Reply:
Yes, 2^n, exactly.